libpod/network/netconflist.go

```diff
@@ -268,3 +255,82 @@ func IfPassesFilter(netconf *libcni.NetworkConfigList, filters map[string][]stri
 	}
 	return result, nil
 }
+
+// ComputeUntilTimestamp extracts unitil timestamp from filters
+func ComputeUntilTimestamp(filter string, filterValues []string) (time.Time, error) {
+	invalid := time.Time{}
+	if len(filterValues) != 1 {
+		return invalid, errors.Errorf("specify exactly one timestamp for %s", filter)
+	}
+	ts, err := timetype.GetTimestamp(filterValues[0], time.Now())
+	if err != nil {
+		return invalid, err
+	}
+	seconds, nanoseconds, err := timetype.ParseTimestamps(ts, 0)
+	if err != nil {
+		return invalid, err
+	}
+	return time.Unix(seconds, nanoseconds), nil
+}
+
+// IfPassesPruneFilter filters NetworkListReport and returns true if the prune filter match the given config
+func IfPassesPruneFilter(config *config.Config, netconf *libcni.NetworkConfigList, f map[string][]string) (bool, error) {
+	for key, filterValues := range f {
+		switch strings.ToLower(key) {
+		case "label":
+			return matchPruneLabelFilters(netconf, filterValues), nil
+		case "until":
```

Comment:
I think it would be good to also add the until filter for network ls

Reply:
It is api extension then. Docker http does not allow this:
https://docs.docker.com/engine/api/v1.41/#operation/NetworkList

Reply:
Let me know if you really want it.

Reply:
I don't thinks its needed when docker doesn't have it and no one asked for it.